projects/ngx-datatable/src/lib/utils/column-helper.ts

```diff
@@ -80,44 +79,3 @@ export function setColumnDefaults(columns: TableColumn[], defaultColumnWidth = 1
 export function isNullOrUndefined<T>(value: T | null | undefined): value is null | undefined {
   return value === null || value === undefined;
 }
-
-/**
- * Translates templates definitions to objects
- */
-export function translateTemplates<TRow>(
-  templates: DataTableColumnDirective<TRow>[]
-): TableColumn[] {
-  const result: TableColumn[] = [];
-  for (const temp of templates) {
-    const col: TableColumn = {};
-
-    const props = Object.getOwnPropertyNames(temp);
-    for (const prop of props) {
-      col[prop] = temp[prop];
-    }
-
-    if (temp.headerTemplate) {
-      col.headerTemplate = temp.headerTemplate;
-    }
-
-    if (temp.cellTemplate) {
-      col.cellTemplate = temp.cellTemplate;
-    }
-
-    if (temp.ghostCellTemplate) {
-      col.ghostCellTemplate = temp.ghostCellTemplate;
-    }
-
-    if (temp.summaryFunc) {
-      col.summaryFunc = temp.summaryFunc;
-    }
```

Comment:
QUick clarification question: Did you on purpose skip copying that property?

Reply:
yes, summaryFunc is not a getter so explicit assignment is not needed.
I don't see any reason why it was there before.